### ISSUE TITLE
Feature Request: Frendly name to cell tower

### ISSUE THREAD

**Issue by olkitu** [CellMapper/CellMapper-Android-App#63]
Is it possible to add a feature that allows us to name sites? This feature makes it easier to identify which site we're connected to. Along with CM I also use the app Signal Check Pro which has this feature and would be nice to stick with one app. It uses a SQLite DB that local to the user but thought CM could use a shared DB that can cache locally and be editable on the app & browser. ﻿

https://plus.google.com/106695696155483821844/posts/EsMbiW6kD9g

**Comment by nelsonjw1** [comment]
The site naming should remain private to the logged-in user and not be displayed publicly to all CellMapper users (or public even to non-logged in users).

**Comment by olkitu** [comment]
@nelsonjw1 Why? Why should not public?

**Comment by nelsonjw1** [comment]
For reasons which I also detailed on Google+. It would be too easy to be abused and made confusing by users. For example, if a user named a tower something such as "Cell tower by my house".  The so-called "Friendly Names" feature would almost certainly be abused, spammed, or made more confusing. There would also be lack of agreement as to local place names or points of reference, as well as the aforementioned subjective naming by using place names or locations known only to the user -- residences, places of employment, and so on.

**Comment by olkitu** [comment]
You are right. Always can spam or something else. Require login adding frendly name so we can see who added and block access if spam

**Comment by brycenesbitt** [comment]
Less potential for abuse: the ability to upload a photo of the tower (only for registered users).  This helps validate an in person manual edit.  Many towers have site numbers visible on signs on the tower.

**Comment by brycenesbitt** [comment]
Also:
Sometimes I might move a tower and I'm certain it's exactly right.  I could document the tower's address or physical site number based on a sign.

Other time's I could only move the tower closer to the right spot, or there's uncertainty.

It would be nice to be able to leave a note to others right in the tower record.

**Comment by olkitu** [comment]
@brycenesbitt We just developing new map and will add new great feature's. We will soon release to BETA Channel. Photo of tower is not just easy because pictures take also space and need system... on old system registered wiki account can add picture + extra information to tower...

**Comment by brycenesbitt** [comment]
Easy add, no pictures:

Tower address____________________________________________
Tower notes_______________________________________________

**Comment by olkitu** [comment]
@brycenesbitt  you think it just easy but it's not just simple :) When input data must have history and abuse system etc... Adding input feature is more complicated than you thinking. On new map you can put coordinate to locate tower, 

Old map have feature to add Note example on "Extra Data" section and also can add picture.

![image](https://user-images.githubusercontent.com/10426342/28337886-b91a1df0-6c0f-11e7-9ccf-4db095092764.png)


Example:

![image](https://user-images.githubusercontent.com/10426342/28337812-8296885e-6c0f-11e7-909d-87beb30670c5.png)

**Comment by brycenesbitt** [comment]
I don't see any feature to add extra per tower data, beyond the wiki:

[image: Inline image 1]

Yes all input needs anti spam, CRSF validation, and connection to an
account (so it can be deleted by account if needed).

**Comment by olkitu** [comment]
@brycenesbitt  i cannot see your image... You need wiki account to edit wiki... But i think we try find better solution to new map.

**Comment by brycenesbitt** [comment]
Here's the image.  No ability to add extra data visible to me:
![cellmaper extra data](https://user-images.githubusercontent.com/1294423/28340575-1bfbf966-6bc5-11e7-9e53-21ca924842c4.png)
A data field visible only to logged in users has less security concern. I would use it when I happen to know the tower's registration number, company site ID, or address from either labels on the site or a government database.  For example in my area I have the building address of all towers nearby.

**Comment by olkitu** [comment]
@brycenesbitt  Click EDIT? And then you can edit with wiki account text and also add picture of tower ....
